src/createAuthScheme.js

```diff
@@ -53,10 +53,22 @@ module.exports = function createAuthScheme(authFun, authorizerOptions, funName,
           type: 'REQUEST',
           path: request.path,
           httpMethod: request.method.toUpperCase(),
-          headers: Object.assign(request.headers, utils.capitalizeKeys(request.headers)),
+          headers: {},
+          multiValueHeaders: {},
           pathParameters: utils.nullIfEmpty(pathParams),
-          queryStringParameters: utils.nullIfEmpty(request.query),
+          queryStringParameters: request.query ? utils.normalizeQuery(request.query) : {},
+          multiValueQueryStringParameters: request.query ? utils.normalizeMultiValueQuery(request.query) : {}
```

Comment:
In the authorization event, `queryStringParameters` and `multiValueQueryStringParameters` are empty objects when no values are specified.